doc/release-notes/release-4.sgml

```diff
@@ -351,6 +353,8 @@ This section gives a thorough account of those changes in three categories:
 	<p>Manual squid.conf update may be required on upgrade.
 	<p>Replaced <em>cafile=</em> with <em>tls-cafile=</em> which takes multiple entries.
 	<p>Changed default value of <em>generate-host-certificates</em> to ON.
```

Comment:
This "default to ON" wording will need to be adjusted -- the option no longer has an "on" value. See the corresponding cf.data.pre change request ("Squid actions" vs. "defaults to enabled").